src/snapred/backend/dao/ingredients/GroceryListItem.py

```diff
@@ -12,7 +10,7 @@ class GroceryListItem(BaseModel):
     Holds necessary information for a single item in grocery list
     """
 
```

Comment:
I would reccommend adding a 
```
def builder():
    return GroceryListBuilder()
```

So that code can just import GroceryListItem and do the following:
```
GroceryListItem.builder().nexus().native().using(item.runNumber).build()
```
w/o needing to import the builder separate.

Reply:
This creates a circular-import situation, as `GroceryListItem` then needs to import `GroceryListBuilder`, but `GroceryListBuilder` needs to import `GroceryListItem` :/

Reply:
if you move the import to the method does it still occur?